python/sanlock.c

```diff
@@ -1674,75 +1676,87 @@ initexception(void)
     return excp;
 }
 
-PyMODINIT_FUNC
-initsanlock(void)
+static int
+util_module_init(PyObject* m)
```

Comment:
I don't think we need the util_ namespace for our static helpers.

Reply:
still TODO